integreat_cms/deepl_api/utils.py

```diff
@@ -61,28 +61,11 @@ def get_target_language_key(self, request, target_language):
         :return: target_language_key which is 2 characters long for all languages except English and Portugese where the BCP code is transmitted
         :rtype: str
         """
-        # For some languages, the DeepL client expects the BCP tag instead of the short language code
-        if target_language.slug in ("en", "pt"):
-            # Check if BCP 47 tag is five characters as expected and contains a "-"
-            if len(target_language.bcp47_tag) == 5 and "-" in target_language.bcp47_tag:
-                target_language_key = target_language.bcp47_tag
-            # If not, check if instead of the BCP47 code the slug was entered
-            elif len(target_language.bcp47_tag) == 2:
-                # Set Great Britain as default for English
-                if target_language.bcp47_tag == "en":
-                    target_language_key = target_language.slug + "-GB"
-                # Set Portugal as default for Portugese
-                elif target_language.bcp47_tag == "pt":
-                    target_language_key = target_language.slug + "-PT"
-            else:
-                messages.error(
-                    request,
-                    _("Your bcp code for the selected language seems to be incorrect."),
-                )
-                target_language_key = None
-        else:
-            target_language_key = target_language.slug
-        return target_language_key
+        deepl_config = apps.get_app_config("deepl_api")
+        for code in [target_language.slug, target_language.bcp47_tag]:
+            if code in deepl_config.supported_source_languages:
```

Comment:
```suggestion
            if code in deepl_config.supported_target_languages:
```
@timoludwig shouldn't it be this? (Not sure if those two are the same, but since we already differentiate, just use `target` for consistency's sake)

Reply:
They are indeed not identical, the source languages do not differentiate between the bcp tags...